src/main/java/fm/last/moji/impl/FileUploadOutputStream.java

```diff
@@ -90,12 +92,17 @@ public void close() throws IOException {
           delegate.close();
         } finally {
           try {
-            String message = httpConnection.getResponseMessage();
             int code = httpConnection.getResponseCode();
             if (HttpURLConnection.HTTP_OK != code && HttpURLConnection.HTTP_CREATED != code) {
-              throw new IOException(code + " " + message);
-            } else {
```

Comment:
I did this because: 1. In my experience this debug message is useless and too verbose 2. The code skips response message parsing  in "happy day".

Sure the intention may out of the topic about this PR. I will roll it back if you think it will be better.

(What we are talking about is ```log.debug("Status: HTTP {} - {}", code, message);``` right?)